doc/user_manual/generated/internalRom.tex

```diff
@@ -42,6 +42,163 @@ \subsubsection{NDspline}
       etc) used in the input HistorySet.
   \default{time}
 
+    \item \xmlNode{featureSelection}:
+      Apply feature selection algorithm
+
+      The \xmlNode{featureSelection} node recognizes the following subnodes:
+      \begin{itemize}
+        \item \xmlNode{RFE}:
+          The \xmlString{RFE} (Recursive Feature Elimination) is a feature selection algorithm.
+          Feature selection refers to techniques that select a subset of the most relevant features
```

Comment:
Is it possible to add featureSelection part in the ROM section, instead of adding for each individual ROM?

Reply:
This comes from the automatic script. I can do it manually but it will be "overwritten" by the next person "re-creating" the documentation. @wangcj05 please advice on how to proceed

Reply:
I see, it is fine to me as it is.